### PR TITLE
API's to add more information for companies (entity) clients

### PR DESCRIPTION
**Pull request by Deepika1095**
https://fiterio.atlassian.net/browse/INKO-61

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by jumba2010 — DISMISSED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        </addColumn>
+    </changeSet>
+    <changeSet author="deepika@fiter.io" id="add_unique_constraints_for_co_signors">
+        <addUniqueConstraint
+                columnNames="co_signors"
+                constraintName="co_signors_constraint"
+                tableName="m_client_other_info"/>
+    </changeSet>
+    <changeSet  author="deepika@fiter.io"  id="dropNotNullConstraint">
+        <dropNotNullConstraint
```
Comment (hero78) [inline fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml:64]: add pre-condition to check if coumn exists before dropping this constraint

File: fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                                columnDataType="int"
+                                columnName="nationality_cv_id"
+                                tableName="m_client_other_info"/>
+        <dropNotNullConstraint
```
Comment (hero78) [inline fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml:68]: same thing here too

**Review by hero78 — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml (hunk 2 above)
Comment (Deepika1095) [reply fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml:68]: fixed

File: fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml (hunk 1 above)
Comment (Deepika1095) [reply fineract-provider/src/main/resources/db/changelog/tenant/custom-changelog/inko_61_add_column_for_client_other_info_company.xml:64]: Fixed

**Review by hero78 — APPROVED** [review]

**Review by jumba2010 — APPROVED** [review]

**Pull request merged**